bokeh/models/widgets/markups.py

```diff
@@ -28,6 +28,26 @@ class Paragraph(Markup):
     The height of the block in pixels.
     """)
 
+class Div(Markup):
+    """ A block (div) of text.
+
+    """
+
+    text = String(default="", help="""
+    The contents of the widget.
+    """)
+
+    width = Int(500, help="""
```

Comment:
I went ahead and changed that.  Based on this comment, should the default width of the Paragraph class also be changed to 600.  I was using that as a model.